src/extension/ip_extraction.c

```diff
@@ -220,6 +222,41 @@ zend_string *nullable dd_ip_extraction_find(
     return NULL;
 }
 
+void dd_ip_extraction_clean_request_values(void)
```

Comment:
This seems to be used only in rshutdown so you might as well merge them into a single function.

Reply:
Done